howitworks.md

```diff
@@ -6,6 +6,18 @@ top_graphic: 3
 redirect_from: "/howitworks"
 ---
 
+# What is Let's Encrypt?
+
+Let's Encrypt allows you to secure your website by concealing the data
+sent and received between your website and the users of your website from prying
+eyes. Prior to the creation of Let's Encrypt, securing your website with SSL/HTTPS
+was often costly and a hassle. Basic security measures shouldn't be a hassle, nor 
+should they be prohibitively expensive in an era where theft of data is becoming
```

Comment:
Let's leave out the "theft of data becoming increasingly common;" we can just end at "prohibitively expensive."